docs/en/sql-reference/functions/date-time-functions.md

```diff
@@ -339,6 +339,30 @@ SELECT toDate('2016-12-27') AS date, toYearWeek(date) AS yearWeek0, toYearWeek(d
 └────────────┴───────────┴───────────┴───────────┘
 ```
 
+## date_trunc(datepart, time\_or\_data\[, time\_zone\]) {#date_trunc}
+
+Truncates a date or date with time based on the specified datepart, such as
+- `second`
+- `minute`
+- `hour`
+- `day`
+- `week`
+- `month`
+- `quarter`
+- `year`
+
+```sql
+SELECT date_trunc('hour', toDateTime('2020-08-20 10:10:10')) AS truncated_date
```

Comment:
I think `dateTrunc ` would be better name as all of other Clickhouse functions are camel case `toDateTime`, `toDateTime64` etc, similar to @hczhcz 's recommendation. `date_trunc` can be an alias if you would like to keep the convention similar to postgres.